src/main/java/uk/gov/hmcts/reform/civil/callback/DashboardWithParamsCallbackHandler.java

```diff
@@ -31,6 +31,7 @@ protected Map<String, Callback> callbacks() {
 
     protected abstract String getScenario(CaseData caseData, CallbackParams callbackParams);
 
+    @SuppressWarnings("java:S1172")
     protected boolean shouldRecordScenario(CaseData caseData) {
```

Comment:
as above

Reply:
Same as above the method has overrides which actually use the method arg